libdevice/atomic.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include "device.h"
 
-#ifdef __SPIR__
+#ifdef __SYCL_SUPPORTED_SPIR_TARGET__
```

Comment:
I'm strongly against adding new macro. We can use existing defines.
```suggestion
#if defined(__SPIR__) || defined(__SPIRV__)
```
E.g. https://github.com/intel/llvm/blob/dc9c91e50dcd4dc7e55d1d79a8b725c0be5aab65/clang/lib/Headers/opencl-c-base.h#L16

Reply:
I considered that but I assumed this was better to prevent the same copy-paste everywhere. Okay, I'll add the or condition.

Reply:
Done in latest commit, PR not published yet tho.